kubernetes/samples/charts/ingress-per-domain/README.md

```diff
@@ -4,7 +4,7 @@ This chart is for deploying an Ingress resource in front of a WebLogic domain cl
 ## Prerequisites
```

Comment:
Traeafik (typo)  -> Traefik 
Change Ingress to lowercase ingress throughout the doc.
Should this sentence include Nginx? "We support two Ingress types: Traefik and Voyager." (In the WKO docs, we have this, "Most common ingress controllers, for example Traefik, Voyager, and NGINX,")

Reply:
done